src/torchcodec/samplers/_implem.py

```diff
@@ -0,0 +1,138 @@
+import random
```

Comment:
random nit: why not call it implementation.py? It's not user-facing, so we can have long descriptive names here

Reply:
I personally don't find `implementation` substantially more descriptive than `implem`.
I'm OK to address, but only at the very end before merging, because the renaming would cause GitHub to mark ongoing comments as "outdated".